handbook/engineering/developer-insights/frontend-platform/index.md

```diff
@@ -118,30 +118,47 @@ The team makes use of [tracking issues](../../tracking_issues.md) for tracking p
 
 Specific product feedback about well-defined, small features can be found directly in the backlog boards. More general product feedback that applies to larger features, or that needs more research and planning to be actionable, is kept in [Productboard](https://sourcegraph.productboard.com/feature-board/2330177-web-frontend-platform)
 
+### Code reviews
```

Comment:
Moved the `Code reviews` section up to have all the meeting-related sections grouped.